content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,184 @@
+
+  .. example::
+     For example, if the group *Sales/Administrator* lists the group *Website/Restricted Editor* in
+     its :guilabel:`Inherited` tab, then any users added to the *Sales/Administrator* group
+     automatically receive access to the *Website/Restricted Editor* group, as well.
+
+- :guilabel:`Menus` tab: defines which menus/models the group can have access to. Click
+  :guilabel:`Add a line` to add a specific *menu*.
+- :guilabel:`Views` tab: lists which views in Odoo the group has access to. Click :guilabel:`Add a
+  line` to add a *view* to the group.
+- :guilabel:`Access Rights` tab: lists the first level of rights (models) that this group has access
+  rights to. Click :guilabel:`Add a line` to link *access rights* to this group. In this tab, the
+  :guilabel:`Model` column represents the common name of the menu/model, and the :guilabel:`Name`
+  column represents the technical name given to the model. For each model, enable the following
+  options as appropriate:
+
+  - :guilabel:`Read`: users can see the object's existing values.
+  - :guilabel:`Write`: users can edit the object's existing values.
+  - :guilabel:`Create`: users can create new values for the object.
+  - :guilabel:`Delete`: users can delete values for the object.
+
+  .. tip::
+     First try searching for the common name of the model in the drop-down menu of the
+     :guilabel:`Model` column. The :guilabel:`Model` technical name can be found by expanding the
+     model common name, which can be done by clicking the :guilabel:`expand` button, represented by
+     an :guilabel:`external link` icon.
+
+     The model technical name can also be accessed in :ref:`developer mode <developer-mode>`.
+
+     On a form, navigate to any field, and hover over the field name. A box of backend information
+     reveals itself with the specific Odoo :guilabel:`Object` name in the backend. This is the
+     technical name of the model that should be added.
+
+     .. image:: access_rights/technical-info.png
+        :align: center
+        :alt: Technical information shown on a field of a model, with object highlighted.
+
+- :guilabel:`Record Rules`: lists the second layer of editing and visibility rights.
+  :guilabel:`Record Rules` overwrite, or refine, the group's *Access Rights*. Click :guilabel:`Add a
+  line` to add a *record rule* to this group. For each rule, choose values for the following
+  options:
+
+  - :guilabel:`Apply for Read`.
+  - :guilabel:`Apply for Write`.
+  - :guilabel:`Apply for Create`.
+  - :guilabel:`Apply for Delete`.
+
+  .. important::
+     *Record rules* are written using a *Domain*, or conditions that filter data. A domain
+     expression is a list of such conditions. For example:
+     `[('mrp_production_ids', 'in', user.partner_id.commercial_partner_id.production_ids.ids)]`.
+     This :guilabel:`Record Rule` is to enable *MRP Consumption Warnings for Subcontractors*.
```

Comment:
Remove formatting and restructure. I don't see a reason for "MRP consumption warnings for subcontractors" to be title-cased or italicized
```suggestion
     Record rules are written using a *domain*, or conditions that filter data. A domain expression
     is a list of such conditions. For example:

     `[('mrp_production_ids', 'in', user.partner_id.commercial_partner_id.production_ids.ids)]`

     This record rule is to enable MRP consumption warnings for subcontractors.
```